src/unity/python/turicreate/CMakeLists.txt

```diff
@@ -49,6 +49,15 @@ set_property(DIRECTORY PROPERTY INSTALLATION_EXTENSIONS "${INSTALLATION_EXTENSIO
 set_property(DIRECTORY PROPERTY INSTALLATION_BINARY_TARGETS "${INSTALLATION_BINARY_TARGETS}")
 set_property(DIRECTORY PROPERTY INSTALLATION_BINARY_FILES "${INSTALLATION_BINARY_FILES}")
 
+add_custom_target(
```

Comment:
I thought we already had this custom target. How did this work prior to this PR?